src/SourceBuild/content/repo-projects/installer.proj

```diff
@@ -10,6 +10,8 @@
 
     <!-- Restore and Build actions are already passed in by the root script. -->
     <BuildActions>$(FlagParameterPrefix)pack $(FlagParameterPrefix)publish</BuildActions>
+    
+    <SkipEnsurePackagesCreated>true</SkipEnsurePackagesCreated>
```

Comment:
Remove this property. This isn't necessary anymore, nor respected.

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/0540025c557a98c47a62c7f24d8a2335c20d179a